src/azure-iot-edge-installer.sh

```diff
@@ -100,33 +100,79 @@ source utils.sh
 log_init
 handlers_init
 
+function show_help() {
+    echo ""
+    echo "${BOLD}Usage: sudo ./azure-iot-edge-installer.sh [OPTION]...${DEFAULT}"
+    echo ""
+    echo "${BOLD}Basic:${DEFAULT}"
+    echo -e "\t-h, --help\t\t\t\t\tPrint this help"
+    echo ""
+    echo "${BOLD}Connection String Provisioning:${DEFAULT}"
+    echo -e "\t-c, --connection-string <CONNECTION_STRING>\tThe Azure IoT Edge Device Connection String"
+    echo -e "${MAGENTA}\tThe default provisioning method. For example,"
+    echo -e "\t$ sudo ./azure-iot-edge-installer.sh -c \"<Connection String>\""
+    echo -e "${DEFAULT}"
+    echo "${BOLD}DPS Provisioning:${DEFAULT}"
+    echo -e "\t-s, --scope-id <SCOPE_ID>\t\t\tThe Azure DPS ID Scope"
```

Comment:
>Azure DPS [](http://example.com/codeflow?start=50&length=9)

nit: I think 'IoT' is missing - Azure IoT DPS

Reply:
fixed - ty